### DISCUSSION TITLE
Installing coreos with ignition file

### DISCUSSION THREAD

**Discussion by rajasaur** [utmapp/UTM#5062] [UTM Help (macOS)]
**Describe the issue**  
Install a Fedora CoreOS Live image and try running the installer to run it on disk rather than memory. The installation process finishes but it doenst seem to install it on disk as when I reboot, it again only boots from memory (Live Disk)

**Configuration**  
* UTM Version: Version 4.1.5 (74) 
* macOS Version: 13.1 (22C65) - Ventura 
* Mac Chip (Intel, M1, ...): Intel Macbook Pro

**Crash log**  
There are no crashes

The CoreOS Installation method is 
`coreos-installer install /dev/sda -I <path to ignition url>`
This downloads the ignition file, runs the installer and finishes without any errors, but when it boots back, it should use the disk to boot, not from memory.

If anyone has tried a CoreOS install and run an ignition file, even a basic create a user ignition file, that would be helpful. Thanks!

**Comment by TommyTran732** [comment]
This is not a problem with Fedora CoreOS specifically. You just need to shut the VM down, remove the liveCD, and boot it up again.

My own butane and ignition files can be found here: https://github.com/tommytran732/Fedora-CoreOS-Ignition (Obviously, don't use it as is - change the user + ssh-keys at least)
